aeolus/aeolus.cpp

```diff
@@ -52,32 +52,46 @@ Synthesizer* createAeolus()
 //   Aeolus
 //---------------------------------------------------------
 
-Aeolus::Aeolus()
+Aeolus::Aeolus() : Synthesizer()
       {
-      model = 0;
+      model = nullptr;
       patchList.append(new MidiPatch { false, "Aeolus", 0, 0, 0, "Aeolus" });
 
       _sc_cmode = 0;
       _sc_group = 0;
       _running = false;
+      _hold = 0;
       _nplay = 0;
-      _fsamp = 0;
       _nasect = 0;
       _ndivis = 0;
+      _revsize = 0.0f;
+      _revtime = 0.0f;
       nout = 0;
-      _ifc_init = 0;
-      for (int i = 0; i < NGROUP; i++)
-            _ifelms [i] = 0;
+      _fsamp = 0.0f;
+      _fsize = 0;
+      _ifc_init = nullptr;
+      memset(_midimap, 0, sizeof(_midimap));
```

Comment:
Isn't there a C++ way of initializing arrays? Like `_midimap = { 0 };` Same for the next couple lines.

Reply:
Ah, that would be my C roots on display again.  In my defense, the constructor already contained one memset, and one loop over the elements of an array.  I just added more memsets so that all of the arrays were zeroed.  I will convert them to this form.

Reply:
Yes, I see that you learned from bad examples there ;-)